### DISCUSSION TITLE
How Can I Run nvim-tree And netrw?

### DISCUSSION THREAD

**Discussion by minusf** [nvim-tree/nvim-tree.lua#1752] [Q&A]
### Description

the docs "strongly advise" disabling netrw but this disables `scp://`, `rsync://` and all the other remote editing possibilities and i consider this a bug.

i love nvim-tree but it would be great if it could coexist peacefully with netrw.

### Neovim version

```text
NVIM v0.8.1
Build type: Release
LuaJIT 2.1.0-beta3
Compiled by brew@HMBRW-A-001-M1-004.local
```

### Operating system and version

macos 12.6.1

### nvim-tree version

9d241e5

### Minimal config

```lua
{}
```

### Steps to reproduce

1. `let loaded_netrwPlugin = 1`
2. `nvim rsync://...`
3. editing empty file...

**Comment by alex-courtis** [comment]
Don't set `vim.g.loaded_netrw` / `vim.g.loaded_netrwPlugin` and nvim-tree will attempt to coexist with netrw.

The following default settings will attempt co-existence:
`:help nvim-tree.disable_netrw`
`:help nvim-tree.hijack_netrw`
`:help nvim-tree.hijack_directories`

It is advised to disable netrw simply because functionality sometimes breaks e.g. race conditions between netrw and nvim-tree on startup.

**Comment by alex-courtis** [comment]
https://github.com/neovim/neovim/issues/4378#issuecomment-1320999806
